src/setup/rgw.py

```diff
@@ -504,8 +504,8 @@ def _logrotate_generic(conf):
         # '/etc/logrotate.d/ceph' -> '/etc/logrotate.d/radosgw'
         # currently component_name is 'rgw', file='/etc/logrotate.d/radosgw'
         old_file = os.path.join(LOGROTATE_DIR, 'ceph')
```

Comment:
Please rephrase the comment above this change.

Reply:
done